scripts/fmt.sh

```diff
@@ -11,4 +11,6 @@ gofmt -w                                                                      \
     $ROOT/grepip                                                              \
     $ROOT/cidr2range                                                          \
     $ROOT/range2cidr                                                          \
-    $ROOT/range2ip
+    $ROOT/range2ip                                                            \
+    $ROOT/cidr2ip
+    
```

Comment:
Add consistent ending newline here (github shows there's some issues apparently).

Reply:
Same for `scripts/lint.sh`.